### DISCUSSION TITLE
APOE: regression

### DISCUSSION THREAD

**Discussion by krootesmurdy** [trendscenter/coinstac#1499] [General]
error from running 2 sites from 2 machines (this file was mapped correctly and visible):

"Pipeline error from pipeline 6335db381b282a216c65b889 user: Emory\n Error details: Computation failed with exitcode 1 and stderr Traceback (most recent call last):\n File \"/computation/scripts/local.py\", line 187, in <module>\n sys.stdout.write(local_0(PARAM_DICT))\n File \"/computation/scripts/local.py\", line 36, in local_0\n covar_x = average_nifti(args)\n File \"/computation/scripts/nipype_utils.py\", line 66, in average_nifti\n image_data = nib.load(os.path.join(input_dir, image)).dataobj[:]\n File \"/usr/local/lib/python3.7/site-packages/nibabel/loadsave.py\", line 107, in load\n raise ImageFileError(f'Cannot work out file type of \"{filename}\"')\nnibabel.filebasedimages.ImageFileError: Cannot work out file type of \"/input/626add4f52a3cb24c42329dd/6335db381b282a216c65b889/9989-swc1Re.nii\"\n"

**Comment by hvgazula** [comment]
Either the file is corrupt, or the correct permissions are not set.

**Reply by rssk** [reply]
This could also be possibly a docker issue, what OS?
